selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NewWorkspace.java

```diff
@@ -200,17 +705,146 @@ public void typeToFiltersInput(String value) {
     seleniumWebDriverHelper.setValue(filterStackInput, value);
   }
 
+  public void deleteLastTagFromInputTagsField() {
+    seleniumWebDriverHelper.waitAndSendKeysTo(filterStackInput, BACK_SPACE.toString());
+    seleniumWebDriverHelper.sendKeys(BACK_SPACE.toString());
+  }
+
+  public void waitFiltersFormOpened() {
+    seleniumWebDriverHelper.waitVisibility(filterStackInput);
+  }
+
+  public void waitFiltersFormClosed() {
+    seleniumWebDriverHelper.waitInvisibility(filterStackInput);
+  }
+
   public String getTextFromFiltersInput() {
     return seleniumWebDriverHelper.waitVisibilityAndGetValue(filterStackInput);
```

Comment:
Actually we can simplify the code a bit if we will use `seleniumWebDriverHelper.waitTextContains()` instead of `getTextFromFiltersInput()` in other methods like `waitTextEqualsInFiltersInput()`.